Example/Tests/Tests.m

```diff
@@ -53,24 +53,25 @@ - (SCORStreamingAnalytics *)create;
     __block Class scorAnalyticsClassMock;
     __block SCORStreamingAnalytics *mockStreamingAnalytics;
     __block SEGComScoreIntegration *integration;
-    __block SCORStreamingPlaybackSession *mockPlaybackSession;
-    __block SCORStreamingAsset *mockAsset;
+    __block SCORStreamingConfiguration *mockConfiguration;
+//    __block SCORStreamingPlaybackSession *mockPlaybackSession;
+//    __block SCORStreamingAsset *mockAsset;
 
     beforeEach(^{
         scorAnalyticsClassMock = mockClass([SCORAnalytics class]);
         mockStreamingAnalytics = mock([SCORStreamingAnalytics class]);
-        mockPlaybackSession = mock([SCORStreamingPlaybackSession class]);
-        mockAsset = mock([SCORStreamingAsset class]);
+//        mockPlaybackSession = mock([SCORStreamingPlaybackSession class]);
```

Comment:
Should these lines be commented or removed?

Reply:
Will remove these as well!